### PR TITLE
New method get_firewall_policies

### PR DESCRIPTION
**Pull request by awlx**
Implementation of https://github.com/napalm-automation/napalm-base/pull/136 for FortiOS

### CODE REVIEW

**Comment by awlx** [comment]
Resolved all conflicts.

**Comment by mirceaulinic** [comment]
For the ```is_alive``` method I can see you provided tests - could you please provide also some code under ```napalm_fortios/fortios.py``` that returns the actual connection state?

**Comment by awlx** [comment]
It's already merged. ;) https://github.com/napalm-automation/napalm-fortios/pull/25

But because of change to the new test framework the tests failed afterwards.

**Comment by mirceaulinic** [comment]
Oh - missed that, sorry.

Thanks @awlx!

**Comment by mirceaulinic** [comment]
Would you be able to have a look at https://travis-ci.org/napalm-automation/napalm-fortios/builds/179704123 and come with a fix (another PR)? The build is failing which is bizarre as this build passed https://travis-ci.org/napalm-automation/napalm-fortios/builds/179704123.

File: test/unit/mocked_data/test_get_firewall_policies/normal/expected_result.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{'1': [{'src_zone': u'any', 'log': u'False', 'service': u'ALL', 'schedule': u'always', 'l3_src': u'all', 'enabled': False, 'packet_hits': -1, 'l3_dst': u'all', 'dst_zone': u'any', 'action': u'permit', 'position': 1, 'byte_hits': -1, 'id': u'1'}], '2': [{'src_zone': u'port3', 'log': u'all', 'service': u'ALL', 'schedule': u'always', 'l3_src': u'all', 'enabled': False, 'packet_hits': -1, 'l3_dst': u'all', 'dst_zone': u'port2', 'action': u'reject', 'position': 2, 'byte_hits': -1, 'id': u'2'}]}
```
Comment (mirceaulinic) [inline test/unit/mocked_data/test_get_firewall_policies/normal/expected_result.json:1]: This is not valid JSON, indeed!

**Review by mirceaulinic — COMMENTED** [review]
Please update ```test/unit/mocked_data/test_get_firewall_policies/normal/expected_result.json``` with a valid JSON and under ```test/unit/mocked_data/test_get_bgp_neighbors/normal/expected_result.json``` add the following content:

```json
{"global": {"router_id": "193.235.32.243,", "peers": {"10.15.2.101": {"is_enabled": true, "uptime": 50292, "remote_as": 65208, "description": "", "remote_id": "74.201.254.126", "local_as": 65432, "is_up": true, "address_family": {"ipv4": {"sent_prefixes": 666, "accepted_prefixes": 1, "received_prefixes": 0}, "ipv6": {"sent_prefixes": 0, "accepted_prefixes": 0, "received_prefixes": 0}}}, "10.15.2.105": {"is_enabled": true, "uptime": 864000, "remote_as": 65209, "description": "", "remote_id": "209.92.66.34", "local_as": 65432, "is_up": true, "address_family": {"ipv4": {"sent_prefixes": 668, "accepted_prefixes": 1, "received_prefixes": 0}, "ipv6": {"sent_prefixes": 3, "accepted_prefixes": 0, "received_prefixes": 0}}}, "10.15.2.99": {"is_enabled": false, "uptime": 0, "remote_as": 65307, "description": "", "remote_id": "0.0.0.0", "local_as": 65432, "is_up": false, "address_family": {"ipv4": {"sent_prefixes": 0, "accepted_prefixes": 0, "received_prefixes": 0}, "ipv6": {"sent_prefixes": 0, "accepted_prefixes": 0, "received_prefixes": 0}}}}}}
```

**Pull request merged**